### DISCUSSION TITLE
2024 Update

### DISCUSSION THREAD

**Discussion by magiccodingman** [magiccodingman/Magic.IndexedDb#6] [Announcements]
For those wondering where I've been, I'm sorry! I created this project out of necessity for another project that's very important to me. This was sidelined when I didn't need the offline capabilities moving forward. Not that I wouldn't need it later, but I no longer needed it in 2023. As of 2024, it's finally back in scope for me to continue this project. I'll be working on some of the bug reports and a major feature I wish to add is my current idea of making the database update process significantly easier and controlled through C#. 

But if anyone wants to take the code, fork it, or provide their own changes, I have zero issues with such things! Sorry for being MIA, but I do still think this project is really cool and don't wish to drop it.

**Comment by vincejairam** [comment]
Does the current release work with .Net 7 ? If so any issues with using .net 8 later on?

**Reply by magiccodingman** [reply]
No issue, and a contributor helped update the to the newest NET version :)
